shell/renderer/atom_renderer_client.cc

```diff
@@ -84,7 +84,7 @@ void AtomRendererClient::DidCreateScriptContext(
           switches::kNodeIntegrationInSubFrames);
   bool should_load_node =
       is_main_frame || is_devtools || allow_node_in_subframes;
-  if (!should_load_node) {
+  if (!should_load_node || IsWebViewFrame(context)) {
```

Comment:
This condition should be added to `should_load_node`.  i.e. `IsWebViewFrame` implies `!should_load_node`